cgroup/cgroup.go

```diff
@@ -59,6 +65,7 @@ func (m manager) read(path string) (string, error) {
 // write appends the given data to the given cgroup file path
 // NOTE: depending on the cgroup file, the append will result in an overwrite
 func (m manager) write(path, data string) error {
+	m.log.Infow("writing to cgroup file", "path", path, "data", data)
```

Comment:
I am happy to turn these down to debug, if desired?

Reply:
I think it'd be okay to merge it as it is, and deactivate it once we're more confortable with the new cgroup pathing